common/baselines/librispeech/ls960/gmm/baseline_args.py

```diff
@@ -0,0 +1,457 @@
+        "tdp_scale": 1.0,
+        "tdp_transition": (3.0, 0.0, "infinity", 0.0),  # loop, forward, skip, exit
+        "tdp_silence": (0.0, 3.0, "infinity", 20.0),
+        "tying_type": "global",
+        "nonword_phones": "",
+        "tdp_nonword": (
+            0.0,
+            3.0,
+            "infinity",
+            6.0,
+        ),  # only used when tying_type = global-and-nonword
+    }
+
+    costa_args = {"eval_recordings": True, "eval_lm": True}
+
+    feature_extraction_args = {
+        "mfcc": {
+            "num_deriv": 2,
+            "num_features": None,  # confusing name: number of max features, above number -> clipped
+            "mfcc_options": {
+                "warping_function": "mel",
+                # to be compatible with our old magic number, we have to use 20 features
+                "filter_width": filter_width_from_channels(
+                    channels=20, warping_function="mel", f_max=8000
+                ),
+                "normalize": True,
+                "normalization_options": None,
+                "without_samples": False,
+                "samples_options": samples_options,
+                "cepstrum_options": {
+                    "normalize": False,
+                    "outputs": 16,  # this is the actual output feature dimension
+                    "add_epsilon": not dc_detection,  # when there is no dc-detection we can have log(0) otherwise
+                    "epsilon": 1e-10,
+                },
+                "fft_options": None,
+                "add_features_output": True,
+            },
+        },
+        "energy": {
+            "energy_options": {
+                "without_samples": False,
+                "samples_options": samples_options,
+                "fft_options": None,
+            }
+        },
+    }
+
+    scorer_args = {"sctk_binary_path": SCTK_BINARY_PATH}
+
+    return util.RasrInitArgs(
+        costa_args=costa_args,
+        am_args=am_args,
+        feature_extraction_args=feature_extraction_args,
+        scorer_args=scorer_args,
+    )
+
+
+def get_monophone_args(feature_flow: str = "mfcc+deriv+norm") -> util.GmmMonophoneArgs:
+
+    linear_alignment_args = {
+        "minimum_segment_length": 0,
+        "maximum_segment_length": 6000,
+        "iterations": 5,
+        "penalty": 0,
+        "minimum_speech_proportion": 0.7,
+        "save_alignment": False,
+        "keep_accumulators": False,
+        "extra_merge_args": None,
+        "extra_config": None,
+        "extra_post_config": None,
+    }
+
+    monophone_training_args = {
+        "name": "mono",
+        "feature_flow": feature_flow,
+        "feature_energy_flow_key": "energy," + feature_flow,
+        "align_iter": 75,
+        "splits": 10,
+        "accs_per_split": 2,
+        "dump_alignment_score_report": True,
+    }
+
+    monophone_recognition_args = {
+        # GmmSystem.recognition() args:
+        "iters": [10],
+        "lm_scales": [10],
```

Comment:
Is this scale close to optimum? I think I remember that GMMs go with LM scales closer to the 20s..

Edit: all other scales are 25 so maybe
```suggestion
        "lm_scales": [25],
```